CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+
+
+
+## Reporting Issues
+
+Have you identified a reproducible problem? Have a feature request? We want to hear about it! Here's how you can make reporting your issue as effective as possible.
+
+### Look For an Existing Issue
+
+Before you create a new issue, please do a search in [open issues](https://github.com/Microsoft/vscode-arduino/issues) to see if the issue or feature request has already been filed.
+
+Be sure to scan through the [most popular](https://github.com/Microsoft/vscode-arduino/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc) issues.
+
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment:
+
+* 👍 - upvote
+* 👎 - downvote
+
+Subscribe to such issues so that you are kept up to date about the progress.
+
+If you cannot find an existing issue that describes your bug or feature, create a new issue using the guidelines below.
+
+### Writing Good Bug Reports and Feature Requests
+
+File a single issue per problem and feature request. Do not enumerate multiple bugs or feature requests in the same issue.
+
+Do not add your issue as a comment to an existing issue unless it's for the identical input. Many issues look similar, but have different causes.
+
+The more information you can provide, the more likely someone will be successful reproducing the issue and finding a fix.
+
+Please include the following with each issue:
+
+* Version of VS Code and this extension
+
+* OS/Platform you are working on
+
+* Reproducible steps (1... 2... 3...) that cause the issue
+
+* What you expected to see, versus what you actually saw
+
+* Images, animations, or a link to a video showing the issue occurring
+
+* A code snippet that demonstrates the issue or a link to a code repository the developers can easily pull down to recreate the issue locally
+
+  * **Note:** Because the developers need to copy and paste the code snippet, including a code snippet as a media file (i.e. .gif) is not sufficient.
+
+* Errors from the Dev Tools Console (open from the menu: Help > Toggle Developer Tools -> Console)
+
+### Final Checklist
+
+Please remember to do the following:
+
+* [ ] Search the issue repository to ensure your report is a new issue
+
+* [ ] Simplify your code around the issue to better isolate the problem
+
+Don't feel bad if others can't reproduce the issue right away. They will simply ask for more information!
+
+### Follow Your Issue
+
+Once submitted, your report will go into the [issue tracking](https://github.com/Microsoft/vscode/wiki/Issue-Tracking) workflow. Be sure to understand what will happen next, so you know what to expect, and how to continue to assist throughout the process.
+
+## Automated Issue Management
```

Comment:
Remove this  section, as we don't want have this kind of bot to close bugs :)

Reply:
Cool! removed that part!

Reply:
thanks a lot !